Examples/JavaScript/JavaScript Essentials/Objects and Arrays.ts

```diff
@@ -0,0 +1,96 @@
+//// { order: 1, compiler: { strict: false } }
```

Comment:
these comments appear in the buffer, which is not bad but a little confusing.

Reply:
It won't eventually 👍